### ISSUE TITLE
Service not able to start

### ISSUE THREAD

**Issue by ntropia2** [bk138/droidVNC-NG#179]
If you'd like to put out an incentive for fixing this bug, you can do so at https://issuehunt.io/r/bk138/droidVNC-NG

**Describe the bug**
The app does not start after assigning all permissions.

 

**To Reproduce**

1. Provide permissions for Input and Notifications
2.  "Screen capturing" stays grayed out as "unknown", but the start button is active.
![Screenshot_20231123_094116_droidVNC-NG](https://github.com/bk138/droidVNC-NG/assets/5226519/75e9081a-1a88-4303-9445-f1b1ece08b5d)
3. press the Start button, and a confirmation is shown to allow the app to start casting:
![Screenshot_20231123_094051_Finder](https://github.com/bk138/droidVNC-NG/assets/5226519/bb5d22ee-b4aa-4102-9e21-8441932fab5f)
4. accept by pressing "Start now"
5.  the service is not started

**Expected Behavior**

The service should start.

**Your environment (please complete the following information):**
 - droidVNC-NG version: 2.1.5
 - Android version: 13, Galaxy S20 FE 5G
 - Client-side OS and version: Kubuntu 23.10
 - VNC client and version: Xvncviewer 1:1.3.10-7

**Comment by bk138** [comment]
The app was tested successfully on an S21FE :thinking: 

* Can you provide logcat output?
* Does https://gitlab.com/vijai/screenrecorder work on the device?

**Comment by ntropia2** [comment]
Took me a while but I finally had the time to test it.

And this is odd. I've tried ScreenRecorder and it worked fine, but DroidVNC had the same behavior (Screen Capturing: UNKNOWN).

Then I've installed and configured LogCat, and when opened the VNC server two things changed:
- input permissions were revoked
- screen capturing was granted

I reissued the input permissions (Accessibility) and it worked. 

No idea why or how that fixed it since I've tried multiple things before this (including rebooting), but this did it. The only thing that changed from earlier was that I had enabled the remote debugging port (`adb tcpip 5555`) while developer mode was already enabled. Again, no idea, but following this procedure somehow fixed it.

Thanks for looking into it, hopefully this might help someone else with similar issues.
